### DISCUSSION TITLE
xgterm Installation issue in Fedora 36

### DISCUSSION THREAD

**Discussion by astro-fermi** [iraf-community/iraf#234] [Q&A] [answered]
Dear all,

I was able to successfully install iraf-2.17 in my Fedora-36 OS. However, when I tried installing x11iraf, I got the following error:

make -C obm
make[1]: Entering directory '/home/astro/Softwares/iraf-2.17/x11iraf-2.1/obm'
cc -I/usr/include/tcl   -c -o Obm.o Obm.c
In file included from Obm.c:5:
ObmP.h:22:10: fatal error: X11/Xaw/Cardinals.h: No such file or directory
   22 | #include <X11/Xaw/Cardinals.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.

Please take a look at the problem and let me know how to solve it. Thank you.

**Comment by olebole** [comment, marked as answer]
Are you sure that the dependent libraries and header files are installed? https://iraf-community.github.io/x11iraf lists
* xaw, 
* xaw3d,
* xmu,
* ncurses, 
* Tcl 
* Xpm

Specifically, this files is an Xaw library header; therefore you should make sure that the Xaw development library and header files are installed (I don't know how they are called in Fedora; maybe *libXaw-devel*).

**Reply by astro-fermi** [reply]
Yes, I had to manually install Xaw, libXaw-devel, Xpm and tcl packages. Once finished, I was able to compile xgterm without any problem. Thank you for helping with this.
